container-images/candlepin/Makefile

```diff
@@ -1,5 +1,5 @@
 build:
-	podman build -t quay.io/ehelms/candlepin:${VERSION} .
+	podman build -t quay.io/ehelms/candlepin:nightly .
```

Comment:
Maybe this is for a follow on to figure out our tagging structure -- cause I think versioned tags alongside nightly tag would be useful.

Reply:
one issue closed, one opened, that's how it goes? :)

Reply:
in some totally unrelated project, I have a `make version` that I then re-use to generate the tag: https://github.com/evgeni/unifi-controller/blob/devel/.github/workflows/release.yml